### PR TITLE
Add deejay de service

### PR DESCRIPTION
**Pull request by TEdwardJ**

### CODE REVIEW

File: src/main/java/com/vinylteam/vinyl/util/ParserConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+public class ParserConfiguration {
+    String vinylGenresSelector;
+    String offerLinkSelector;
+    String releaseSelector;
+    String artistSelector;
+    String catalogNumberSelector;
+    String priceDetailsSelector;
+    String highResolutionImageSelector;
+    String inStockMarkerSelector;
+    String inStockMarker;
```
Comment (AlexKievDeveloper) [inline src/main/java/com/vinylteam/vinyl/util/ParserConfiguration.java:18]: Please add empty row after class declaration and before closing, according to common codestyle.

File: src/main/java/com/vinylteam/vinyl/util/RawOffersSorter.java [hunk 2]
```diff
@@ -50,24 +50,24 @@ void addOffersSortingByVinyl(List<RawOffer> rawOffers, UniqueVinyl uniqueVinyl,
             int percentMatching = 75;
             String uniqueVinylRelease = uniqueVinyl.getRelease();
             char lastCharInRawRelease = uniqueVinylRelease.charAt(uniqueVinylRelease.length() - 1);
-            if (lastCharInRawRelease > '0' && lastCharInRawRelease < '9'){
+            if (lastCharInRawRelease > '0' && lastCharInRawRelease < '9') {
                 percentMatching = 90;
             }
             String[] preparedFullNameForMatching = Arrays.stream(uniqueVinyl.getFullName().split("[- ()!@$%^&*_={}:;\"']")).filter(e -> e.trim().length() > 0).toArray(String[]::new);
             Iterator<RawOffer> rawOfferIterator = rawOffers.iterator();
+            uniqueVinyl.setHasOffers(false);
```
Comment (AlexKievDeveloper) [inline src/main/java/com/vinylteam/vinyl/util/RawOffersSorter.java:58]: :1st_place_medal:

**Review by AlexKievDeveloper — DISMISSED** [review]

**Review by AlexKievDeveloper — DISMISSED** [review]

File: src/main/java/com/vinylteam/vinyl/util/ParserConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+package com.vinylteam.vinyl.util;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.Value;
```
Comment (Yuliia-Sh) [inline src/main/java/com/vinylteam/vinyl/util/ParserConfiguration.java:4]: Data is not used here, correct?

**Review by Yuliia-Sh — DISMISSED** [review]

**Review by Greengrer — COMMENTED** [review]
Already commented on all that on another pull rquest, please take a look.

File: src/main/java/com/vinylteam/vinyl/util/ParserConfiguration.java (hunk 3 above)
Comment (TEdwardJ) [reply src/main/java/com/vinylteam/vinyl/util/ParserConfiguration.java:4]: Yes, thanks. To be fixed

**Review by AlexKievDeveloper — APPROVED** [review]

**Review by Yuliia-Sh — APPROVED** [review]

**Pull request merged**